build/commands/lib/config.js

```diff
@@ -402,43 +390,48 @@ Config.prototype.buildArgs = function () {
   let versionParts = version.split('+')[0]
   versionParts = versionParts.split('.')
 
+  const is_component_build = this.isComponentBuild() ? true : undefined
+  const is_official_build = this.isOfficialBuild() ? true : undefined
+
   let args = {
     'import("//brave/build/args/brave_defaults.gni")': null,
-    is_asan: this.isAsan(),
```

Comment:
moved to https://github.com/brave/brave-core/pull/30844/files#diff-4a7ab71a64ac5712bb35fb91782847039cf2def6ee9c3d64a7efa66562a99acfR421